app/routes/entries/browse/$initialLetters/$pageNumber.tsx

```diff
@@ -19,11 +23,20 @@ export default function EntryDetailsPage() {
   const data = useLoaderData<typeof loader>()
   const params = useParams()
 
+  let curPage: number;
+
+  if (params.pageNumber) {
+    curPage = parseInt(params.pageNumber);
+  } else {
+    curPage = 1;
+  }
+
```

Comment:
This can all go on one line using the ternary operator:

```ts
const currentPage = params.pageNumber ? parseInt(params.pageNumber) : 1
```

(I also prefer to spell out entire words in variable names — just makes it that much easier to read the code, in my experience.)

Reply:
I also just want to point out (you may well already know it) that there's a potential gotcha with this kind of code, if you ever consider 0 to be a valid value. `params.pageNumber` is falsy if pageNumber is 0, even though 0 is a valid integer. In cases like that you'll want to do an additional check that allows 0:

```ts
const currentPage = params.pageNumber || params.pageNumber === 0 ? ... : ...
```

In this case we don't consider 0 to be a valid value, so it's fine.

Reply:
oh yeahh that's a great point-- I didn't think of that, good to know going forward

Reply:
Done